package.json

```diff
@@ -78,7 +78,7 @@
     "cross-env": "7.0.2",
     "css-loader": "3.4.2",
     "electron-devtools-installer": "3.0.0",
-    "electronite": "7.2.1-graphite",
+    "electronite": "^12.0.5-beta",
```

Comment:
Should we add the beta version?

Reply:
that was an accident. I published 12.0.5 to npm but there was a bug in it. And NPM won't allow you to re-use a deployed version. So it has to be `12.0.5-[something]`. I could deploy it again as `12.0.5-graphite` so it seems more official if you think that is necessary.